examples/pom.xml

```diff
@@ -32,7 +32,7 @@ under the License.
 
   <modules>
     <module>beam</module>
-    <module>spark</module>
+<!--    <module>spark</module>-->
```

Comment:
I think this change must have gotten in there by mistake, let's revert this change.

Reply:
Changed. Thanks!